src/ast/nodes/SequenceExpression.ts

```diff
@@ -65,15 +68,15 @@ export default class SequenceExpression extends NodeBase {
 		);
 	}
 
-	includeInBundle() {
-		let addedNewNodes = !this.included;
+	include() {
+		let anotherPassNeeded = false;
 		this.included = true;
 		for (let i = 0; i < this.expressions.length - 1; i++) {
 			const node = this.expressions[i];
-			if (node.shouldBeIncluded() && node.includeInBundle()) addedNewNodes = true;
+			if (node.shouldBeIncluded() && node.include()) anotherPassNeeded = true;
```

Comment:
Perhaps this loop also can break out early if one condition matches? Likely won't be an issue if `this.expressions` is a small array.

Reply:
As stated above, this is a bad idea.